_includes/homepage/announcements.html

```diff
@@ -0,0 +1,53 @@
+<section class="bp-section {{ gray_container_class }}">
+  <div class="bp-container is-fluid">
+    <div class="row">
+      <div class="col is-full p-16">
+        {%- if section.announcements.subtitle -%}
+        <p class="eyebrow is-uppercase pb-4 has-text-centered">
+          {{- section.announcements.subtitle -}}
+        </p>
+        {%- endif -%} {%- if section.announcements.title -%}
+        <h1 class="has-text-secondary pb-4 has-text-centered">
+          <b>{{- section.announcements.title -}}</b>
+        </h1>
+        {%- endif -%}
+
+        <hr class="my-8 announcements-divider" />
+
+        {%- for announcement in section.announcements.announcement_items -%}
+        <div class="row is-desktop announcements-announcement-row-px-desktop">
```

Comment:
what happens on mobile views?

Reply:
clarified offline - only the `min-width` part isn't applied

Reply:
Yup, by default `col` will stack on top of one another unless `is-desktop` is specified.